features/steps/api_steps.py

```diff
@@ -109,3 +122,9 @@ def i_can_see_the_ping_information(context):
     i_see_revision_in_response(context)
     i_see_release_id_in_response(context)
     i_see_commit_id_in_response(context)
+
+
+@then("I can see the prescription in the response")
```

Comment:
This is where our approaches differ slightly. Happy with either way tbh. Have a look at the prescribe and cancel assertions on mine and let me know what you think.